Dockerfile.aarch64

```diff
@@ -15,14 +15,20 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:xenial
+FROM arm64v8/debian:stretch
```

Comment:
@justincormack this is most likely a result of the arm folks requesting image metadata being more specific regarding ISA/ABI from discussion in the image-spec / at dockercon https://github.com/opencontainers/image-spec/issues/661.